### ISSUE TITLE
no-missing-blank-lines false positives

### ISSUE THREAD

**Issue by davidosomething** [wooorm/remark-lint#49]
for lists using double indent

```
- list item
    - nested item with 4 space indent
```

the rule expects a blank line in between the two
I believe this is a false positive since the block nodes are nested, not adjacent when rendered as html

**Comment by wooorm** [comment]
The reason this rule warns is because there’s no newline between the paragraph and the list, which are adjacent to each other. Both are block nodes, so in my opinion this rule is right to warn here!
